src/bzflag/GUIOptionsMenu.cxx

```diff
@@ -366,7 +347,7 @@ void            GUIOptionsMenu::resize(int _width, int _height)
 
     // use a big font for title, smaller font for the rest
     const float titleFontSize = (float)_height / 15.0f;
-    const float fontSize = (float)_height / 65.0f;
+    const float fontSize = (float)_height / 70.0f;
```

Comment:
is there anything to be gained by being consistent? In **effectsMenu** this is 60.

Reply:
The whole menu font size thing is a bit wonky.  We don't have any sort of automatic scaling depending on how many menu items are in a menu so we just tweak the number a bit to make it look okay.